bql/semantic/expression.go

```diff
@@ -81,6 +81,15 @@ func (o OP) String() string {
 	}
 }
 
+// cellFromRow retrives the value of a binding from a given row.
```

Comment:
Little typo here:

s/retrives/retrieves

Reply:
Thank you Thiago, I have just corrected it.